flutter-studio/src/io/flutter/startup/FlutterStudioInitializer.java

```diff
@@ -0,0 +1,34 @@
+package io.flutter.startup;
+
+import com.intellij.ide.actions.NewProjectAction;
+import com.intellij.openapi.actionSystem.ActionManager;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.startup.StartupActivity;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.projectImport.ProjectOpenProcessor;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+
+public class FlutterStudioInitializer implements StartupActivity {
+
+  @Override
+  public void runActivity(@NotNull Project project) {
+    NewProjectAction newProject = new NewProjectAction();
+    newProject.getTemplatePresentation().setText("New &Project...", true);
+    newProject.getTemplatePresentation().setDescription("Create a new project from scratch");
+    replaceAction("NewProject", newProject);
```

Comment:
Can you add a TODO: here? (my understanding is that replacing the project wizard is useful for us in the short-term, but not something we'd want to ship)